src/main/java/org/kainos/ea/resources/DeliveryEmployeeController.java

```diff
@@ -13,25 +13,26 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-@Api("Evian Week 2 Exercise Delivery Employees API")
-@Path("/api")
+
+@Api("Evian Week 2 Exercise Employees API")
+@Path("/api/employees")
 public class DeliveryEmployeeController {
 
     private DeliveryEmployeeService deliveryEmployeeService = new DeliveryEmployeeService();
     @GET
-    @Path("/delivery_employees")
+    @Path("/delivery")
     @Produces(MediaType.APPLICATION_JSON)
     public List<Employee> getEmployees() {
         Object deliveryEmployeeService;
```

Comment:
What's the `deliveryEmployeeService` `Object` being used for?

Reply:
place holder for now

Reply:
changes made